src/screens/savings/index.tsx

```diff
@@ -0,0 +1,4 @@
+import React, { FC } from 'react';
+import { Text } from 'react-native';
+
+export const Savings: FC = () => <Text>TODO: Add some contents</Text>;
```

Comment:
do we need this TODO for now?

Reply:
I think we need it to prevent crashing on accidental card click

Reply:
ok, no problem